src/coreclr/scripts/genRuntimeEventSources.py

```diff
@@ -147,6 +155,13 @@ def generateEvent(eventNode, providerNode, outputFile, stringTable):
     outputFile.write(")\n")
     writeOutput(outputFile, "{\n")
 
+    increaseTabLevel()
+    # Wrap the call to WriteEvent with IsEnabled check
```

Comment:
If I understood correctly all the events need to be hand-edited before they will handle Lttng correctly? If that is the case I think it would be much safer to generate a method body that throws NotImplementedException() and includes a comment describing what a future dev will need to change if they want it to have a correct implementation.

There is also a more minor concern that adding new auto-generated in the method body is making the size of System.Private.CoreLib.dll bigger storing code that will never be invoked.

Reply:
Ok, that makes sense. I'll modify this to throw instead :)